src/polyglot-notebooks-vscode-common/src/metadataUtilities.ts

```diff
@@ -286,7 +298,7 @@ export function getRawInteractiveDocumentMetadataFromNotebookDocumentMetadata(no
     return notebookDocumentMetadata;
 }
 
-export function getRawNotebookDocumentMetadataFromNotebookDocumentMetadata(notebookDocumentMetadata: NotebookDocumentMetadata, createForIpynb: boolean): { [key: string]: any } {
+export function getRawNotebookDocumentMetadataFromNotebookDocumentMetadata(notebookDocumentMetadata: NotebookDocumentMetadata, documentRawMetadata: { [key: string]: any }, createForIpynb: boolean): { [key: string]: any } {
```

Comment:
It seems that this method is now doing more than one responsibility. Perhaps something like getAndMergetNotebookMetadata

Reply:
good point, renamed